### DISCUSSION TITLE
Is it normal that the NDH from SO are creating security alerts for normal traffic?

### DISCUSSION THREAD

**Discussion by b-u-g-s** [Security-Onion-Solutions/securityonion#8687] [General] [answered]
Hi,

I just installed a security onion honeypot, it is in the same network as my SO instance.
I noticed a lot of security alerts related to the honeypot, but not because anyone is accessing it, it looks like normal admin/maintenance traffic.

I can whitelist those type of alerts, but it would be great if SO was already aware that this is part of the grid, it is a NDH and thus there is no need to generate alerts for "normal" traffic.

Below are the alerts that are flogging in:
ET POLICY GNU/Linux YUM User-Agent Outbound likely related to package management
Rule ID: 2013505

Cheers,
B.

**Comment by dougburks** [comment, marked as answer]
> Is it normal that the NDH from SO are creating security alerts for normal traffic?

Yes, this is normal. We intentionally do not whitelist NIDS alerts for the grid. In the unlikely event that an attacker is able to gain access to a node in the grid, we want to see any and all alerts.

You are of course welcome to tune your ruleset in any way you see fit. That might be suppressing all alerts for the IP address or suppressing that SID for your grid. Some organizations even disable the entire POLICY category. It's all up to you.

**Reply by b-u-g-s** [reply]
Thanks Doug, make sense
